services/booking-api/helpers/makeBookingRequest.js

```diff
@@ -0,0 +1,15 @@
+import * as request from '../../../libs/request';
+
+export default async (url, apikey, body) => {
+  try {
+    const requestClient = request.requestClient(
+      { rejectUnauthorized: false },
+      { 'X-ApiKey': apikey }
+    );
+
+    const response = await request.call(requestClient, 'post', url, body);
+    return response;
+  } catch (error) {
+    throw new Error(error);
+  }
+};
```

Comment:
I think you can skip the try-catch block here and instead use the await-to-js to handle the promise and throw error if it occurs.

```JS
async function makeBookingRequest(url, apiKey, body) {
   const requestClient = request.requestClient(
        { rejectUnauthorized: false },
        { 'X-ApiKey': apikey }
    );

    const [response, error] = await to(request.call(requestClient, 'post', url, body));
    if (error) {
       throw error
    }
    return response;
}
```

Reply:
Would also consider renaming the function/name to something mor explicit. A name that represents the intention or expected outcome for the function, ie what form of request is this? Is it a creation of a singel booking or multiple ones, or something else? :)

Reply:
Having a hard time to naming this function to a more specific one. Since this function is making requests against Datatorget with multiple outcomes, i.e. creating/cancel/get bookings. So the outcome differs depending on which url and body that it receives.

Reply:
Then i think sendBookingPostRequest is sufficient maybe? Or simply have helper functions that use this function ie sendCancelBookingRequest, sendCreateBookingRequest etc :)

You could look at the viva-ms/helpers/vivaRequestClient for some inspiration in how to setup this and export the requests towards the datatorget api.

Reply:
Let's start by only changing the name to a little more specific one. If we don't think that's enough in the future then I suggest we change it then and keep it as simple as possible for now.